package-lock.json

```diff
@@ -3979,6 +3979,14 @@
       "integrity": "sha1-w2GT3Tzhwu7SrbfIAtu8d6gbHA8=",
       "dev": true
     },
+    "monotone-convex-hull-2d": {
```

Comment:
Note that I was only replying to the fact that the lock file should be reviewed. I have my answer regarding those changes and I'm OK to have them in this PR.

…well, provided they are in a commit on their own (as in #413), so +1 to getting #413 merged before this PR :wink: